StRoot/StBFChain/BigFullChain.h

```diff
@@ -1363,6 +1363,15 @@ Bfc_st BFC[] = { // standard chains
   {"gstarLib","","",""                                                 ,"","gstar","Load gstar lib",kFALSE},
   {"flux"        ,"","","simu"                                           ,"","flux","Load flux lib",kFALSE},
   {"------------","-----------","-----------","------------------------------------------","","","",kFALSE},
+  {"Generators  ","-----------","-----------","------------------------------------------","","","",kFALSE},
```

Comment:
DId not intend to commit this, but it is a harmless addition to our list of chain options (and will be utilized as we merge in the G4star codes).

Reply:
If I understand correctly, if you update the branch from which you are making the PR not to have this line, then it will drop off the list of changed files. Then you can re-apply this change later when it is needed.

Reply:
Can you clarify if it is only the single "Generators" line that isn't needed for this commit, or all the BigFullChain.h line changes in this PR that aren't really needed for this PR? git makes my approval mandatory for the whole shebang of this PR because I'm a code owner for a small part of it :-(